core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -353,7 +353,9 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
       consumeRecords(this.consumers.head)
       Assert.fail("should have thrown exception")
     } catch {
-      case e: TopicAuthorizationException => assertEquals(Collections.singleton(topic), e.unauthorizedTopics());
+      case e: TestTimeoutException => //expected
+    } finally {
+      this.consumers.foreach(_.wakeup())
```

Comment:
You explained this in another test, probably worth having the same comment here.